.gitignore

```diff
@@ -122,6 +123,7 @@ venv.bak/
 
 # python venv installed in the dir, llmfoundry-venv
 *-venv
+*-env
```

Comment:
```suggestion
```
is this a venv name you personally use?
I feel like most ppl just use `venv` or something more specific.